test/rest_request_template/exactly_once_semantic_time_buffer.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Adding a different file for time based threshold otherwise it will use same topic and SF table and data will already be present. (Hence duplicates will be detected)